log/Logger.js

```diff
@@ -1,18 +1,49 @@
 const util = require('util');
 const {createLogger, format, transports} = require('winston');
-const {errors, json, metadata, timestamp} = format;
+const {errors, json, metadata, prettyPrint, printf, splat, timestamp} = format;
+
+const DEFAULT_LOG_LEVEL = 'info';
+const NPM_LOG_LEVEL_MAP = {
+  error: 'error',
+  warn: 'warn',
+  info: 'info',
+  debug: 'debug',
+  trace: 'verbose'
+}
 
 const customMessageFormat = format((info) => {
-  const {message, metadata} = info;
+  const {message: infoMsg, metadata, ...otherInfo} = info;
+  let message = infoMsg;
   if (metadata && typeof metadata === 'object' && Object.keys(metadata).length) {
-    info.message = message + util.inspect(metadata);
+    message = message + util.inspect(metadata);
   }
-  // delete metadata as its concatenated to the message
-  delete info.metadata;
 
-  return info;
+  return {
+    message,
+    ...otherInfo
+  };
 });
 
+const logLevel = () => {
+  const level = NPM_LOG_LEVEL_MAP[(process.env.LOG_LEVEL || DEFAULT_LOG_LEVEL).toLowerCase()];
+  return level ? level : DEFAULT_LOG_LEVEL;
+};
+
+const logFormat = () => {
+  const logFormat = (process.env.LOG_FORMAT || 'json').toLowerCase();
```

Comment:
`json` could be extracted to a `DEFAULT_LOG_FORMAT` const.

Reply:
done